### DISCUSSION TITLE
DeArrow only the title, leave the native thumbnail

### DISCUSSION THREAD

**Discussion by WildGenius** [libre-tube/LibreTube#5636] [Ideas]
### Describe your suggested feature

The desktop DeArrow version allows to choose between replacing thumbnails, titles, or both. I'd like to replace only the titles in LibreTube.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
I currently don't see the necessity of this, hence concerting to a discussion for now.

**Reply by Kommynct** [reply]
The original thumbnails contain much more useful information than the dearrow ones, the dearrow titles contain much more useful information than the originals. You get the best of each this way.

**Comment by WildGenius** [comment]
No necessity, just feature parity with the desktop version and less image requests. Plus, in my experience, the thumbnails are more likely to contain actually useful information.
